### ISSUE TITLE
[MHV Meds / Accessibility] Focused items obscured by expanded print/download menu

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#123446]
## Issue Description
This is a sibling to the same issue in Medical Records.  Details need to be updated to reflect Medication base... links and screenshot.

### 📝 User story
As a keyboard user, I want the item that currently has keyboard focus to be completely visible.

### 📝 Problem statement
Focused items are obscured when the print/download menu remains expanded, even after focus moves outside of the component.

### 📝 Details
<details><summary>Links, screenshots, steps to recreate</summary>

#### 🌌 Supporting screenshots, videos, code snippets

---

##### 🔗 From [Medical Records > Allergies and reactions](https://staging.va.gov/my-health/medical-records/allergies)

<img width="670" height="262" alt="Screenshot showing a partially visible focused element under an expanded print/download menu" src="https://github.com/user-attachments/assets/d7d36501-a941-4012-9e0f-25af0d662ac7" />

---

#### 👣 Steps to recreate
1. Navigate to the Medications > Allergies and reactions page.
2. Use the Tab key to navigate to the Print or download actions menu component; select Enter to expand the menu.
3. Use the Tab key to move focus through the menu items.
4. Use the Tab key to move focus to the next focusable element after the Print or download actions menu component
5. Notice that the focus indicator (yellow border) is outlining an element that is under the expanded menu component.
</details>

---

### 🔗 Point of contact
Sarah Horton (@sarahhortonadhoc)

### ⚠️ Severity level
- 2, Serious. Should be fixed in 1-2 sprints post-launch.

#### Accessibility Standards
- [Guideline 2.4 Navigable: Provide ways to help users navigate, find content, and determine where they are](https://www.w3.org/WAI/WCAG21/Understanding/navigable)
  - [SC 2.4.11 Focus Not Obscured (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps
- Close the actions button menu component when focus moves out of the component.

### 🔎 Supporting resources
- [Focus management](https://design.va.gov/accessibility/focus-management)

### ✅ Acceptance criteria
- [ ] The system closes the print/download actions menu button when focus moves outside of the component.
- [ ] The user can see complete focusable elements and their complete focus indicator when they have focus.

### ✅ Possible tasks
- [x] Review, refine and acknowledge feedback.
- [x] Update ticket information (assignee, labels, project(s), etc.)
- [x] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [x] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

### 🔎 General resources

- USWDS Docs: [USWDS Documentation](https://designsystem.digital.gov/)
- VADS Docs: [VADS Experience Standards](https://design.va.gov/about/experience-standards/)
- VADS Docs: [VADS Components Documentation](https://design.va.gov/components/)
- VADS Docs: [VADS Components Storybook](https://design.va.gov/storybook/?path=/docs/about-introduction--docs)
- VA Platform Docs: [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
- VA Github Docs: [List of common issue tickets](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility/common-issues) you can reference, copy, and add to.
- W3C Docs: [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
- W3C Docs: [WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)

**Comment by steveg-IAT** [comment]
@Dottifacilities, just wanted to give you a heads-up that the shared support team will address this accessibility issue in the coming sprint.  How do you want to track that on your end? I can add it to your board in the monitor swinlane. I will be in the Fri sync meeting to provide an update, but let me know if you have any questions or concerns before that. 
cc. @sarahhortonadhoc

**Comment by steveg-IAT** [comment]
20251126:
 just getting back to this ticket today...

- updated failing tests for error alert rendering and block messaging

Draft PR going in, pending "Shai-Hulud 2.0 Supply Chain Attack"
